var/spack/repos/builtin/packages/draco/package.py

```diff
@@ -41,12 +41,6 @@ class Draco(CMakePackage):
     version("6.20.1", sha256="b1c51000c9557e0818014713fce70d681869c50ed9c4548dcfb2e9219c354ebe")
     version("6.20.0", sha256="a6e3142c1c90b09c4ff8057bfee974369b815122b01d1f7b57888dcb9b1128f6")
 
-    variant(
-        "build_type",
-        default="Release",
-        description="CMake build type",
-        values=("Debug", "Release", "RelWithDebInfo", "MinSizeRel"),
-    )
```

Comment:
If the default is truly `Release`, this deletion is correct.  For Draco, `build_type=RelWithDebInfo` does not have the normal meaning and users will find that the build behaves in unexpected ways.  The default build must be `Release`.